spec/pdc/v1/multi_destination_spec.rb

```diff
@@ -27,5 +27,10 @@
     it 'return empty data for specific global component and origin repo release' do
       multi_destinations.where(active: true, global_component: 'ceph', origin_repo_release_id: 'ceph-1.3').all!.must_equal []
     end
+
+    it 'return real data with query params' do
+      mappings = multi_destinations.where(active: true, global_component: 'ceph', origin_repo_release_id: 'rhceph-2.1@rhel-7').all!
+      mappings.map(&:destination_repo).map(&:release_id).uniq.sort.must_equal ['ceph-2.0@rhel-7', 'rhceph-2.1-updates@rhel-7']
+    end
```

Comment:
seems uniq can also remove from here
But this looks good to me

Reply:
Do you mean according to the test data, this uniq can be removed here?
When I write the case, I am assuming there is the case that some duplicate release_ids exist, so that is the test data problem, Ack, updated the test data.